cloudbuild.yaml

```diff
@@ -0,0 +1,20 @@
+steps:
+  # FRONTEND STEPS
```

Comment:
Not sure what this is supposed to mean

Reply:
Copy and paste error from a file I used as a starting template, thank you for pointing it out I will clean up the comment.